test-data/simple/request.yaml

```diff
@@ -0,0 +1,102 @@
+inputs:
+  - id: test
+    source:
+      localFile:
+        path: /test/input.txt
+steps:
+  - stage: 0
+    action:
+      executeCommand:
+        sandboxName: compile
+        cwd: /
+        argv:
+          - g++
+          - /prog.cpp
+          - -o
+          - /out/bin
+        env: []
+        stdio:
+          stdin: dev-null
+          stdout: compile-stdout
+          stderr: compile-stderr
+  - stage: 0
+    action:
+      createFile:
+        id: compile-stdout
```

Comment:
Not readable at all. Should be changed (if possible) to something like:
id: file: compile-stdout

Reply:
File part `createFile` implies that id will identify a file.